main/src/main/scala/sbt/TemplateCommandUtil.scala

```diff
@@ -177,6 +177,7 @@ private[sbt] object TemplateCommandUtil {
     "holdenk/sparkProjectTemplate.g8" -> "A Scala Spark project",
     "spotify/scio.g8" -> "A Scio project",
     "disneystreaming/smithy4s.g8" -> "A Smithy4s project",
+    "softwaremill/tapir.g8" -> "A tapir-based application using Netty and upickle"
```

Comment:
I am +1 on adding this, but I'd like to make the description as short as possible since long string gets messed up on the terminal window.

```suggestion
    "softwaremill/tapir.g8" -> "A tapir project using Netty"
```

Also assuming it's a Play competitor, I'd suggest moving it directly under Play to line 176.

Reply:
Fixed, thanks!